src/wiki/useful-windows-tips.md

```diff
@@ -242,6 +242,60 @@ Computer\HKEY_CURRENT_USER\Software\Classes\Local Settings\Software\Microsoft\Wi
+
+1. Open Control Panel. (Tip: Press <kbd>Win</kbd> + <kbd>R</kbd> and type `control` to quickly open it.)
+
+2. Head to Power Options. From there, click "Choose what the power button does". Click "Change settings that are currently unavailable", uncheck Turn on Fast Startup, then click Save Changes.
+
+![Fast Startup](./img/useful-windows-tips/fastboot.png)
+
+## Reducing disk space usage
+
+Every so often you'd find yourself in a pinch when the C drive runs low on space and goes red. Fortunately, Windows packs some tools to make more space, which are easily user-accessible granted they have administrator privileges.
+
+1. The good old GUI utility, Disk Cleanup. Do remember to click "Clean up system files", since these temporary files can grow large enough and turn into a burden.
+
+   ![Disk Cleanup](./img/useful-windows-tips/cleanmgr.png)
+
+2. Reducing hiberfile size. The hibernation file is responsible for hibernation and Fast Startup, and it scales with the total system memory, but you can significantly reduce its size while still keeping its features.
+
+   In an admin command prompt, type `powercfg /h /size 40` to reduce the hiberfile size to 40% of its original size, which is the minimum required for hibernation support.
+
+   ![Hiberfile 40%](./img/useful-windows-tips/pwrcfg40.png)
+
+   If you do not wish to use hibernation, but still want to use Fast Startup (e.g. you have an HDD as your boot drive) then use `powercfg /h /type reduced` instead, which will set the hiberfile size to 20% of its original size (see the difference). Note that this disables hibernation.
+
+   ![Hiberfile reduced](./img/useful-windows-tips/pwrcfgredux.png)
+
+   If you don't need Fast Startup either, use `powercfg /h off` to completely disable hibernation, fast startup, and delete the hibernation file.
+
+3. Compact OS (applicable to Windows 10 and above)
+
+   Compact OS is a neat feature, which compresses Windows system files. Its predecessor, called WIMBoot, was introduced in Windows 8, as Microsoft's first attempt at reducing the Windows image footprint, but proved to be problematic in the case of system updates and recovery. It was superseded by Compact OS in Windows 10. Compact OS can be used with both legacy BIOS and UEFI systems.
+
+   In an admin command prompt, use `compact /compactos:always` to enable Compact OS. If you ever need to get out of the compact state, just use `compact /compactos:never` to uncompress your system files.
+
+   ![Compact OS](./img/useful-windows-tips/compact.png)
+
+4. Cleaning up the WinSxS folder
+
+   The WinSxS is a crucial folder, specifically it is the Windows component store. Sometimes this folder can grow very large and some might want to delete it - but this is highly advised against, since this may render your Windows installation entirely unusable. But while it is not deletable, you can still reduce its size using a simple command (requires administrator permission):
```

Comment:
minor change
```suggestion
   The WinSxS is a crucial folder : it is the Windows component store. Sometimes this folder can grow very large and some might want to delete it - but this is highly advised against, since this may render your Windows installation entirely unusable. But while it is not deletable, you can still reduce its size using a simple command (requires administrator permission):
```